services/gateway_manager/src/test/java/com/futurewei/alcor/gatewaymanager/controller/GatewayControllerTest.java

```diff
@@ -52,16 +51,16 @@ public void createGatewayInfoTest() throws Exception {
 
         mockMvc.perform(post(url_createGatewayInfo).contentType(MediaType.APPLICATION_JSON).content(UnitTestConfig.vpcInfoJson()))
                 .andDo(print())
-                .andExpect(status().isCreated())
-                .andExpect(jsonPath("$.id").value(UnitTestConfig.vpcId));
+                .andExpect(status().isCreated());
+                //.andExpect(jsonPath("$.id").value(UnitTestConfig.vpcId));
```

Comment:
Can we enable this test?

Reply:
The test is enabled, but line 55 will failed when zetaGatewayEnabled=false.  I have modify the code to check zetaGatewayEnabled and perform the test for both case.